README.md

```diff
@@ -1,70 +1,57 @@
-# Getting Started with Create React App
+# Student App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Student App is a full-stack web application that allows users to create, edit, and delete both student and course records. It also allows users to take and drop courses for each student.
 
-## Available Scripts
+## Table of Contents
 
-In the project directory, you can run:
+- Motivation
+- Installation
+- Usage
+- API
+- Tools
+- Authors
 
-### `npm start`
+## Motivation
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+Student App was Inspired by my college's schedule builder. I wanted to create a web application that would allow students manage their courses and their schedule.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Installation
 
-### `npm test`
+Use the package manager npm to install Student App.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```bash
+npm install
+```
 
-### `npm run build`
+## Usage
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+Users has to be registered and logged in to use the application.
+Users can create, edit, and delete both student and course records.
+Users can take and drop courses for each student.
 
```

Comment:
add some more instructions!
Probably something like:

`npm run start`

Should also note that it requirs the API to be running, and a mongodb server running, and to run locally needs the appropriate env variabls set in a `.env` file